### PR TITLE
Update network-config-template to new format

### PR DESCRIPTION
**Pull request by fao89**
ref: https://github.com/openstack-k8s-operators/dataplane-operator/pull/426

### CODE REVIEW

**Review by jistr — APPROVED** [review]

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
recheck

**Review by marios — APPROVED** [review]
we need this to get the job back to green 

we can see it run here if you rebase for the horizon skip and then merge this today

**Comment by arxcruz** [comment]
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'role_networks' is undefined. 'role_networks' is undefined\n\nThe error appears to be in '/home/zuul/src/github.com/openstack-k8s-operators/data-plane-adoption/tests/roles/dataplane_adoption/tasks/main.yaml': line 155, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: deploy dataplane\n  ^ here\n"}

**Comment by marios** [comment]
latest one ... is it something new now @fao89 ? 

        * https://logserver.rdoproject.org/66/166/431f82a2d780b8adc3ec608c412a2626bbe5b736/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/77541f5/controller/pod/dataplane-deployment-configure-network-openstack-hnnsl-logs.txt
``` 
        * [1;31mfatal: [standalone]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.122.100 port 22: No route to host", "unreachable": true}[0m
```
        * https://logserver.rdoproject.org/66/166/431f82a2d780b8adc3ec608c412a2626bbe5b736/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/77541f5/controller/pod/dataplane-deployment-configure-network-openstack-md9r4-logs.txt
```  
      * [1;31mfatal: [standalone]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.122.100 port 22: No route to host", "unreachable": true}[0m
```

**Comment by fao89** [comment]
recheck

**Comment by fao89** [comment]
new failure now:
```
TASK [osp.edpm.edpm_iscsid : Download needed container] ************************
Wednesday 27 September 2023  17:38:36 +0000 (0:00:00.154)       0:00:45.484 *** 
[1;30mFAILED - RETRYING: [standalone]: Download needed container (5 retries left).[0m
[1;30mFAILED - RETRYING: [standalone]: Download needed container (4 retries left).[0m
[1;30mFAILED - RETRYING: [standalone]: Download needed container (3 retries left).[0m
[1;30mFAILED - RETRYING: [standalone]: Download needed container (2 retries left).[0m
[1;30mFAILED - RETRYING: [standalone]: Download needed container (1 retries left).[0m
[0;31mfatal: [standalone]: FAILED! => {"attempts": 5, "changed": false, "msg": "Failed to pull image quay.io/podified-antelope-centos9/openstack-iscsid:current-podified"}[0m
```
https://logserver.rdoproject.org/66/166/7feb72b655cc3cf2d08385e6018a98e091a232e7/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/4a4fc4e/controller/pod/dataplane-deployment-download-cache-openstack-wsnj9-logs.txt

**Comment by abays** [comment]
> new failure now:
> 
> ```
> TASK [osp.edpm.edpm_iscsid : Download needed container] ************************
> Wednesday 27 September 2023  17:38:36 +0000 (0:00:00.154)       0:00:45.484 *** 
> �[1;30mFAILED - RETRYING: [standalone]: Download needed container (5 retries left).�[0m
> �[1;30mFAILED - RETRYING: [standalone]: Download needed container (4 retries left).�[0m
> �[1;30mFAILED - RETRYING: [standalone]: Download needed container (3 retries left).�[0m
> �[1;30mFAILED - RETRYING: [standalone]: Download needed container (2 retries left).�[0m
> �[1;30mFAILED - RETRYING: [standalone]: Download needed container (1 retries left).�[0m
> �[0;31mfatal: [standalone]: FAILED! => {"attempts": 5, "changed": false, "msg": "Failed to pull image quay.io/podified-antelope-centos9/openstack-iscsid:current-podified"}�[0m
> ```
> 
> https://logserver.rdoproject.org/66/166/7feb72b655cc3cf2d08385e6018a98e091a232e7/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/4a4fc4e/controller/pod/dataplane-deployment-download-cache-openstack-wsnj9-logs.txt

Perhaps running out of disk space?  I have been seeing this with my baremetal attempts lately.

**Comment by fao89** [comment]
I'll try temporarily remove `download-cache` to see how it goes

**Comment by fao89** [comment]
Dropping `download-cache` just moved the error:
```
TASK [osp.edpm.edpm_iscsid : Fetch iscsid.conf from the iscsid container] ******
Wednesday 27 September 2023  21:26:32 +0000 (0:00:00.719)       0:00:50.523 *** 
[0;31mfatal: [standalone]: FAILED! => {"changed": true, "cmd": ["podman", "run", "-ti", "--rm", "--name", "iscsid_config", "quay.io/podified-antelope-centos9/openstack-iscsid:current-podified", "cat", "/etc/iscsi/iscsid.conf"], "delta": "0:00:00.257256", "end": "2023-09-27 21:26:33.236737", "msg": "non-zero return code", "rc": 125, "start": "2023-09-27 21:26:32.979481", "stderr": "Trying to pull quay.io/podified-antelope-centos9/openstack-iscsid:current-podified...\nError: initializing source docker://quay.io/podified-antelope-centos9/openstack-iscsid:current-podified: pinging container registry quay.io: Get \"https://quay.io/v2/\": dial tcp: lookup quay.io on [::1]:53: read udp [::1]:35128->[::1]:53: read: connection refused", "stderr_lines": ["Trying to pull quay.io/podified-antelope-centos9/openstack-iscsid:current-podified...", "Error: initializing source docker://quay.io/podified-antelope-centos9/openstack-iscsid:current-podified: pinging container registry quay.io: Get \"https://quay.io/v2/\": dial tcp: lookup quay.io on [::1]:53: read udp [::1]:35128->[::1]:53: read: connection refused"], "stdout": "", "stdout_lines": []}[0m
```
https://logserver.rdoproject.org/66/166/3ab7ab5cbe8363a5f09b640af5b50899e8969dfa/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/1dbf913/controller/pod/dataplane-deployment-configure-os-openstack-d88fp-logs.txt

File: tests/roles/dataplane_adoption/tasks/main.yaml [hunk 1]
```diff
@@ -166,13 +166,9 @@
         - ctlplane
       preProvisioned: true
       services:
-        - download-cache
         - configure-network
         - validate-network
         - install-os
-        - configure-os
-        - run-os
-        - ovn
```
Comment (fao89) [inline tests/roles/dataplane_adoption/tasks/main.yaml:175]: I'm temporarily dropping these services to get the CI green, I'll read them in a follow-up PR

**Comment by fao89** [comment]
CI is back to green now,
I started this: https://github.com/openstack-k8s-operators/data-plane-adoption/pull/171
and we can investigate why it fails to pull quay.io/podified-antelope-centos9/openstack-iscsid:current-podified

**Comment by marios** [comment]
thanks for working on that @fao89

**Pull request merged**